<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog2</class>
 <widget class="QDialog" name="StartDialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>160</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="timeGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Time Control</string>
   </property>
   <widget class="QLabel" name="timeCondition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelByoyomi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Byoyomi</string>
    </property>
   </widget>
   <widget class="QComboBox" name="byoyomiBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="hourBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">6</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="minBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">40</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>hour</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSecond">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>seconds</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StartDialog2</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StartDialog2</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
